consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/MergeProtocolSchedule.java

```diff
@@ -58,6 +61,25 @@ public static ProtocolSchedule create(
         .createProtocolSchedule();
   }
 
+  public static TimestampSchedule createTimestamp(
+      final GenesisConfigOptions config,
+      final PrivacyParameters privacyParameters,
+      final boolean isRevertReasonEnabled) {
+    return new TimestampScheduleBuilder(
+            config,
+            DEFAULT_CHAIN_ID,
+            TimestampProtocolSpecAdapters.create(
+                config.getShanghaiTimestamp().orElse(0),
+                (specBuilder) ->
+                    MergeProtocolSchedule.applyMergeSpecificModifications(
+                        specBuilder, config.getChainId())),
+            privacyParameters,
```

Comment:
I think there's a discussion to be had around whether or not we support timestamps for non-mainnet. I've tried to leave it open for both. Extra code to support it has been minimal and I don't think it's caused any problems so far, other than more 'noise' in the code...maybe warrants a manual test with a private network.

I don't feel too strongly either way, support can easily be added back in later if necessary.

Reply:
I was taking the position that if we can support a feature for private networks then by default we do because I think that's the approach we have been taking. But there probably isn't a lot of value in supporting this for private networks. So if the code is simpler by not supporting private networks then we could remove the timestamp support for private networks.

Reply:
For the `MergeProtocolSchedule` though I can't think of my we would need to have the privacy parameters. For IBFT or QBFT we use a different protocol schedule.

Don't think this is necessary to block the PR though, so could clean this up later.